WordPress/src/main/java/org/wordpress/android/ui/pages/PageItemViewHolder.kt

```diff
@@ -49,29 +60,18 @@ sealed class PageItemViewHolder(internal val parent: ViewGroup, @LayoutRes layou
                 else
                     pageItem.title
 
-                if (pageItem.labels.isEmpty()) {
-                    firstLabel.visibility = View.GONE
-                    smallStretcher.visibility = View.VISIBLE
-                    largeStretcher.visibility = View.GONE
-                } else {
-                    firstLabel.text = parent.context.getString(pageItem.labels.first())
-                    firstLabel.visibility = View.VISIBLE
-                    smallStretcher.visibility = View.GONE
-                    largeStretcher.visibility = View.VISIBLE
-                }
+                val date = if (pageItem.date == Date(0)) Date() else pageItem.date
+                time.text = DateTimeUtils.javaDateToTimeSpan(date, parent.context).capitalize()
```

Comment:
I was going to ask whether it was possible to make this calculation outside of the `ViewHolder`, but it looks like it'd be a pain to do so. As far as I know date calculations can be expensive and I was hoping that it'd be done in a background thread in the view models, but I think it'd be outside of the scope of this PR.

Reply:
We could do this in a separate PR if we find it's needed for the performance. But sites usually don't have that many pages so it probably won't be an issue.

Reply:
Sure, that is fine by me.